### ISSUE TITLE
TensorFlow常用函数（二） | SparkydogX Blog

### ISSUE THREAD

**Issue by sparkydogX** [sparkydogX/sparkydogx_blog_comment#119]
https://sparkydogx.github.io/2017/08/29/tf-function-001/ 

 tf.nn.dropout1234567dropout(    x,    keep_prob,    noise_shape=None,    seed=None,    name=None)  函数名称：tf.nn.dropout 输入参数： x:一个tensor量，通常为一个全连接层 keep_prob：一个tensor标量，表示神经元保持活性的概率 noise_shape：一个1维的int
